src/workerd/jsg/jsg.c++

```diff
@@ -278,6 +278,29 @@ JsSymbol Lock::symbolAsyncDispose() {
   return IsolateBase::from(v8Isolate).getSymbolAsyncDispose();
 }
 
+void Lock::adjustExternalMemory(ssize_t amount) {
```

Comment:
nit: since we static_cast to int64_t, why not have a int64_t parameter? and also why `ssize_t` here but `size_t` in ExternalMemoryAdjuster constructor?

Reply:
`ssize_t` because the `adjustExternalMemory` will take positive or negative values.

`size_t` in the constructor because that only needs to accept positive numbers and we'll invert it internally.

As for why `size_t` and not `int64_t`... really just so it is explicit in the API what the values are intended to be.